### PR TITLE
add real git-repository name to update manager api

### PR DESCRIPTION
**Pull request by FrYakaTKoP**
Add real repository name to the ws msg.

more info see issue #257

### CODE REVIEW

File: moonraker/components/update_manager/git_deploy.py [hunk 1]
```diff
@@ -321,6 +322,14 @@ async def initialize(self, need_fetch: bool = True) -> None:
             self.git_owner = "?"
             if owner_match is not None:
                 self.git_owner = owner_match.group(1)
+
+            # Parse GitHub Repository Name from URL
+            repo_match = re.match(r".*/(.*)\..*", self.upstream_url)
```
Comment (Arksine) [inline moonraker/components/update_manager/git_deploy.py:327]: FWIW, it is possible that the upstream url won't have a ".git" extension, in which case this regex probably won't match.

**Comment by FrYakaTKoP** [comment]
just changed to a better regex matching either with or without the .git extension

Credits to un!t for the regex

**Comment by FrYakaTKoP** [comment]
un!t just came up with yet a shorter regex :)

**Comment by Arksine** [comment]
Looks good, thanks.

**Pull request merged**